.ado/windows-vs-pr.yml

```diff
@@ -27,11 +27,11 @@ jobs:
     condition: ne( dependencies.Setup.outputs['checkPayload.shouldSkipPRBuild'], 'True' )
     strategy:
       matrix:
-        #X64Debug:
-        #  BuildConfiguration: Debug
-        #  BuildPlatform: x64
-        #  applyRnPatches: true
-        #  LayoutHeaders: true
+        X64Debug:
+          BuildConfiguration: Debug
+          BuildPlatform: x64
+          applyRnPatches: true
+          LayoutHeaders: true
```

Comment:
There was an issue I ran into before with re-enabling LayoutHeaders. If you see CI failures, it might be related.

Reply:
Thanks! I commented it out back.

---
In reply to: [375533297](https://github.com/microsoft/react-native-windows/pull/4040#discussion_r375533297) [](ancestors = 375533297)